src/libraries/System.Linq/tests/Shuffler.cs

```diff
@@ -1,14 +1,24 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
 using System.Collections;
 using System.Collections.Generic;
 
 namespace System.Linq.Tests
 {
     public static class Shuffler
     {
+        public static T[] Shuffle<T>(T[] array)
+        {
+            int i = array.Length;
+            while (i > 1)
+            {
+                int j = Random.Shared.Next(i--);
```

Comment:
Maybe we should use a seeded `Random` instance here for deterministic test inputs?

Reply:
Sure.  I'll do that in a separate PR.